.eslintrc.json

```diff
@@ -1,14 +1,16 @@
 {
   "extends": "eslint:recommended",
   "env": {
-    "browser": true
+    "browser": true,
+    "node": true,
+    "es6": true
   },
   "parserOptions": {
     "ecmaVersion": 6
   },
   "rules": {
     "no-unused-vars": 0,
-    "no-undef": 0,
+    "no-undef": "error",
```

Comment:
Think this line can just get deleted since it's in the "recommended" rules https://eslint.org/docs/rules/no-undef